README.md

```diff
@@ -51,8 +53,22 @@ representation _(as described in RFC-4122)_.
 uuid(); // "52e6953d-edbe-4953-be2e-65ed3836b2f0"
 ```
 
-All random values in UUIDs produced by this API must be generated from a **[cryptographically
-secure][csprng]** source.
+## `Math.getRandomValues()`
```

Comment:
Nit: is the headline level on purpose (same as overview but higher level than UUID API)?